packages/elements-core/src/components/Docs/Docs.tsx

```diff
@@ -46,18 +25,40 @@ export interface IDocsComponentProps<T = unknown> {
   uri?: string;
```

Comment:
While we're here, how 'bout renaming this? `uri` means *everything*. It's great that we have JSDoc, but what if we called it like `documentUri` or something, and then the JSDoc could describe what it's used for in the first place.

Reply:
You will have to give me some name though.

Because `documentUri` doesn't tell me much either... :sweat_smile: